SPECS/python-rich/0001-Skip-tests-for-mariner.patch

```diff
@@ -0,0 +1,176 @@
+From 25e012a5ebed83930e84c7c8cf0cd0e0b4e84dee Mon Sep 17 00:00:00 2001
+From: Sam Meluch <sammeluch@microsoft.com>
+Date: Tue, 14 May 2024 17:03:13 -0700
+Subject: [PATCH] Skip tests for mariner
+
+---
+ tests/test_card.py                   | 11 +++++++++++
+ tests/test_markdown.py               | 11 +++++++++++
+ tests/test_markdown_no_hyperlinks.py | 11 +++++++++++
+ tests/test_syntax.py                 |  8 +++++++-
+ 4 files changed, 40 insertions(+), 1 deletion(-)
+
+diff --git a/tests/test_card.py b/tests/test_card.py
+index d578ec89..f1683af2 100644
+--- a/tests/test_card.py
++++ b/tests/test_card.py
+@@ -1,11 +1,21 @@
+ import io
+ import re
++import sys
++import pytest
+ 
+ from rich.__main__ import make_test_card
+ from rich.console import Console, RenderableType
+ 
+ from ._card_render import expected
+ 
++if sys.version_info >= (3, 8):
++    from importlib.metadata import Distribution
++else:
++    from importlib_metadata import Distribution
```

Comment:
Is `else` even possible for Azure Linux 3.0 and up?

Reply:
uhh likely not, I can remove these instances. This is merely a copy paste error from other code within python-rich.

Reply:
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=570211&view=results - new build running with updated patch with else clauses removed.